content/v2.0/reference/flux/stdlib/testing/diff.md

```diff
@@ -34,16 +38,22 @@ _The `testing.diff()` function can be used to perform in-line diffs in a query._
 ## Parameters
 
 ### got
-The stream containing data to test.
+Stream containing data to test.
 _Defaults to piped-forward data (`<-`)._
 
 _**Data type:** Object_
 
 ### want
-The stream that contains the expected data to test against.
+Stream that contains the expected data to test against.
 
 _**Data type:** Object_
 
+### epsilon
+Specifies how far apart two **float** values can be, but still considered equal.
+Defaults to `0.000000001`.
```

Comment:
I would recommend this to become something like `1e-9`, hard to count those zeros.

Reply:
I agree, but Flux doesn't support float values using scientific notation and I don't want to confuse users.

Reply:
I see, thanks!

Reply:
I created https://github.com/influxdata/flux/issues/2732, hoping that we could have scientific notation supported one day.